src/streaming/controllers/PlaybackController.js

```diff
@@ -392,7 +396,7 @@ function PlaybackController() {
         const currentTime = getNormalizedTime();
         const actualTime = getActualPresentationTime(currentTime, mediaType);
         const timeChanged = (!isNaN(actualTime) && actualTime !== currentTime);
-        if (timeChanged && !isSeeking()) {
+        if (timeChanged && isStalled()) {
```

Comment:
Can we keep the seeking check? Otherwise we might seek on top of an ongoing seek. Or does isStalled return false in case we are seeking?

Reply:
Not sure in fact. I prefer indeed keep seeking check.